lib/SILGen/SILGen.cpp

```diff
@@ -1380,10 +1380,12 @@ static bool canStorageUseTrivialDescriptor(SILGenModule &SGM,
     auto setter = decl->getSetter();
     if (setter == nullptr)
       return true;
-    
-    auto setterLinkage = SILDeclRef(setter, SILDeclRef::Kind::Func)
-      .getLinkage(NotForDefinition);
-    
+
+    auto setterLinkage =
+        SILDeclRef(setter, SILDeclRef::Kind::Func)
+            .getLinkage(NotForDefinition,
+                        true /*assume private imports disabled*/);
```

Comment:
Can you explain what the issue here was? If there's a problem with uniquing a trivial descriptor when you reference a private property from another module, I would expect it to happen with -enable-testing as well.

Reply:
Yes that is the problem and it affects -enable-testing as well,  test/stdlib/KeyPathMultiModule.swift will fail with -enable-testing.